### PR TITLE
Increase coverage of is argument function

### PR DESCRIPTION
**Pull request by khamiltonuk**
#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. <your-steps-here>

#### Checklist for author

- [ ] `npm run lint` passes
- [ ] References to standard library functions are [cached](https://github.com/sinonjs/sinon/pull/1523).

### CODE REVIEW

File: lib/is-arguments.js [hunk 1]
```diff
@@ -2,34 +2,31 @@
...
 var getClass = require("./get-class");
 
+// This is for IE11, which can't reliably detect arguments
+// However, it allows use of callee in strict mode without blowing up
+var engineCanDetectArguments = (function test() {
+    return getClass(arguments);
+})();
+
 /**
- * Returns `true` when `object` is an `arguments` object, `false` otherwise
```
Comment (mroderick) [inline lib/is-arguments.js]: Your editor is quite aggressive with the JSDoc, and doesn't understand the settings file :(

**Comment by mantoni** [comment]
Can I ask you to revert the indentation changes?

**Comment by mroderick** [comment]
> Can I ask you to revert the indentation changes?

Kristian and I have been collaborating on this PR, we'll do an interactive rebase to make this read better.

**Comment by mroderick** [comment]
But yeah, the indentation changes are annoying. I imagine that @khamiltonuk hasn't enabled [EditorConfig](https://editorconfig.org) in their IDE, which is why it is using whatever settings are set as default.

File: lib/is-arguments.js (hunk 1 above)
Comment (khamiltonuk) [reply lib/is-arguments.js]: thanks, for spotting this, it now fixed

**Comment by khamiltonuk** [comment]
> > Can I ask you to revert the indentation changes?
> 
> Kristian and I have been collaborating on this PR, we'll do an interactive rebase to make this read better.

My editor was following the `.editorconfig` but, prettier config was overriding it

**Pull request merged**